projects/simulation/human_dataset_generation/README.md

```diff
@@ -0,0 +1,48 @@
+# Human-Data-Generation-Framework
+
+This folder contains the code for generating the data described in "Efficient Realistic Data Generation Framework leveraging Deep Learning-based Human Digitization"
+
+## Download and reformat the CityScapes dataset
+
+1. Download the 3D human models
+
+```
+chmod +x download_models.sh
+./download_models.sh
+```
+
+2. Download the CityScapes dataset from www.cityscapes-dataset.net <br />
+    * RGB images: (a) leftImg8bit_trainvaltest.zip,  (b) leftImg8bit_trainextra.zip <br />
+    * Annotation images: gtCoarse.zip <br />
+
+The folder hierarchy should look like this:
+```
+├─ background_images
+|  ├─ CityScapes
```

Comment:
Could you please just confirm that the folder name is `CityScapes`?

Reply:
I renamed the folder's name to "Cityscapes" in README.md and in the scripts that read/write files from it.